### DISCUSSION TITLE
How to set a timeout in the function websocket_recv_msg of WebSocket subsystem?

### DISCUSSION THREAD

**Discussion by c4explosive** [zephyrproject-rtos/zephyr#57199] [Q&A]
The [documentation](https://docs.zephyrproject.org/latest/connectivity/networking/api/websocket.html#c.websocket_recv_msg) talks about the timeout parameter, but when i put a number in milliseconds (like the docs says), this parameter is not respected or doesn't had any effect on receive routine, the routine is called forever and not return until it have receive some data.

**Comment by jukkar** [comment]
Looks like a bug, please create a bug report about this.

**Reply by c4explosive** [reply]
Thanks for your answer, i created the [bug report here](https://github.com/zephyrproject-rtos/zephyr/issues/57464).
